### PR TITLE
MTM-34850 added docs

### PR DESCRIPTION
**Pull request by DawidSAG**

### CODE REVIEW

File: content/users-guide/cockpit-bundle/dashboards.md [hunk 1]
```diff
@@ -103,3 +103,18 @@ An alternative way to copy a dashboard is to use the
 ### To delete a dashboard
 
 To delete a dashboard from an object, click **More...** in the top menu bar and from the context menu select **Delete dashboard**.
+
+
+### Dashboard availability
+Dashboard availability adds the ability to control which users have access to certain dashboards based on global roles. The functionality is designed to improve dashboard management.
+
+A new section called **Availability** has been added to the edit/create dashboard view. By default, all available global roles are checked, which means that a user with at least one such role will have access to this dashboard.
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/dashboards.md:111]:
```suggestion
In the **Availability** section, specify which users have access to the dashboard based on global roles. By default, all available global roles are selected, which means that a user with at least one such role has access to the dashboard.
```

File: content/users-guide/cockpit-bundle/dashboards.md [hunk 2]
```diff
@@ -103,3 +103,18 @@ An alternative way to copy a dashboard is to use the
 ### To delete a dashboard
 
 To delete a dashboard from an object, click **More...** in the top menu bar and from the context menu select **Delete dashboard**.
+
+
+### Dashboard availability
+Dashboard availability adds the ability to control which users have access to certain dashboards based on global roles. The functionality is designed to improve dashboard management.
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/dashboards.md:109]:
```suggestion
```

File: content/users-guide/cockpit-bundle/dashboards.md [hunk 3]
```diff
@@ -103,3 +103,18 @@ An alternative way to copy a dashboard is to use the
...
+
+A new section called **Availability** has been added to the edit/create dashboard view. By default, all available global roles are checked, which means that a user with at least one such role will have access to this dashboard.
+
+<img src="/images/users-guide/cockpit/cockpit-dashboard-availbility.png" name="Dashboard availability"/>
+
+{{< c8y-admon-important >}}
+Dashboard will always be visible to its owner or if the user has ADMIN permission for permission type "Inventory".
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/dashboards.md:116]:
```suggestion
- Dashboard are always visible to its owner and to users with ADMIN permission for the permission type "Inventory".
```

File: content/users-guide/cockpit-bundle/dashboards.md [hunk 4]
```diff
@@ -103,3 +103,18 @@ An alternative way to copy a dashboard is to use the
...
+
+
+### Dashboard availability
+Dashboard availability adds the ability to control which users have access to certain dashboards based on global roles. The functionality is designed to improve dashboard management.
+
+A new section called **Availability** has been added to the edit/create dashboard view. By default, all available global roles are checked, which means that a user with at least one such role will have access to this dashboard.
+
+<img src="/images/users-guide/cockpit/cockpit-dashboard-availbility.png" name="Dashboard availability"/>
+
+{{< c8y-admon-important >}}
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/dashboards.md:115]:
```suggestion
{{< c8y-admon-info >}}
```

File: content/users-guide/cockpit-bundle/dashboards.md [hunk 5]
```diff
@@ -103,3 +103,18 @@ An alternative way to copy a dashboard is to use the
...
+Dashboard will always be visible to its owner or if the user has ADMIN permission for permission type "Inventory".
+{{< /c8y-admon-info >}}
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/dashboards.md:117]:
```suggestion
```

File: content/users-guide/cockpit-bundle/dashboards.md [hunk 6]
```diff
@@ -103,3 +103,18 @@ An alternative way to copy a dashboard is to use the
...
+{{< c8y-admon-info >}}
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/dashboards.md:118]:
```suggestion
```

File: content/users-guide/cockpit-bundle/dashboards.md [hunk 7]
```diff
@@ -103,3 +103,18 @@ An alternative way to copy a dashboard is to use the
...
+This functionality is based entirely on client-side solutions. If the user has an accurate link to the dashboard, he will still be able to access it.
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/dashboards.md:119]:
```suggestion
- This functionality is entirely based on client-side solutions. If users have an accurate link to the dashboard, they will still be able to access it.
```

Comment (BeateRixen) [reply content/users-guide/cockpit-bundle/dashboards.md:119]: is based entirely on client-side solutions > what does that mean?

**Comment by BeateRixen** [comment]
@DawidSAG The entire text should be integrated into the section "To add a dashboard" and not be a separate section at the bottom.

**Review by BeateRixen — APPROVED** [review]

**Review by eniosultan — APPROVED** [review]

**Pull request merged**